packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -585,6 +599,36 @@ export default class Health extends Component {
     )
   )
 
+  _getNonSharedDefaultSr = createCollectionWrapper(
+    createSelector(
+      () => this.props.hosts,
+      () => this.props.pools,
+      () => this.props.userSrs,
+      () => this._getPoolIds(),
+      (hosts, pools, userSrs, poolIds) => {
+        let _pools = {}
+        const poolsWithMultipleHosts = {}
+
+        if (isEmpty(poolIds)) {
+          _pools = pools
+        } else {
+          forEach(poolIds, poolId => (_pools[poolId] = pools[poolId]))
+        }
+        const nonSharedDefaultSr = filter(_pools, pool => !userSrs[pool.default_SR].shared)
+
+        forEach(hosts, host => {
+          const nbOfHostsInThePool = poolsWithMultipleHosts[host.$pool]
+          if (nbOfHostsInThePool === undefined) {
+            poolsWithMultipleHosts[host.$pool] = 1
+          } else {
+            poolsWithMultipleHosts[host.$pool] = nbOfHostsInThePool + 1
+          }
+        })
```

Comment:
```suggestion
       poolsWithMultipleHosts =  _.countBy(hosts, host => host.$pool)
```
or, if you dont' want to use lodash
```suggestion
    forEach(hosts, host => {
    poolsWithMultipleHosts[host.$pool] = poolsWithMultipleHosts[host.$pool] ?? 0 + 1
    }
```